management-api-server/doc/openapi.json

```diff
@@ -872,6 +872,25 @@
         "summary" : "Move node on the token ring to a new token. This operation returns immediately with a job id."
       }
     },
+    "/api/v0/ops/node/reload-truststore" : {
```

Comment:
issue: This is still different than ``@PATH``

Reply:
This should be auto-generated during the compile phase, so long as you don't specify `-DskipOpenApi`. Either the skip option was used, or you forgot to add this file to your most recent commit

Reply:
Regenerated.